basculehttp/log.go

```diff
@@ -69,11 +72,36 @@ func sanitizeHeaders(headers http.Header) (filtered http.Header) {
 func SetLogger(logger *zap.Logger) alice.Constructor {
 	return func(delegate http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			r = r.WithContext(sallust.With(r.Context(),
-				logger.With(
-					zap.Reflect("requestHeaders", sanitizeHeaders(r.Header)), //lgtm [go/clear-text-logging]
-					zap.String("requestURL", r.URL.EscapedPath()),
-					zap.String("method", r.Method))))
+			var tid string
+			if tid = r.Header.Get(candlelight.HeaderWPATIDKeyName); tid == "" {
+				tid = candlelight.GenTID()
+			}
+
+			var source string
+			host, _, err := net.SplitHostPort(r.RemoteAddr)
+			if err != nil {
+				source = r.RemoteAddr
+			} else {
+				source = host
+			}
+
+			logger = logger.With(
+				zap.Reflect("requestHeaders", sanitizeHeaders(r.Header)), //lgtm [go/clear-text-logging]
+				zap.String("requestURL", r.URL.EscapedPath()),
+				zap.String("method", r.Method),
```

Comment:
Can we make these match the other request related key value pairs? `request.whatever`?